src/constants/common.js

```diff
@@ -123,6 +123,21 @@ module.exports = {
 	INVITED: 'INVITED',
 	ENROLLED: 'ENROLLED',
 	UNIT_OF_TIME: 'minutes',
+	fileTypeCSV: 'text/csv',
+	NO_OF_ATTEMPTS: 3,
+	BACK_OFF_RETRY_QUEUE: 600000,
+	tempFolderForBulkUpload: 'public/invites',
+	sessionOutputFile: 'output-session-creation',
+	csvExtension: '.csv',
+	responseType: 'stream',
+	azureBlobType: 'BlockBlob',
+	UPLOADED_STATUS: 'UPLOADED',
+	FAILED_STATUS: 'FAILED',
+	PROCESSED_STATUS: 'PROCESSED',
```

Comment:
can we have in 
`{
  "STATUS": {
    "FAILED": "FAILED",
    "PROCESSED": "PROCESSED",
    "UPLOADED": "UPLOADED"
  }
}
`
this format.

Reply:
@nevil-mathew Will have to look into it , how it works .